### PR TITLE
Add exchange rate to enabled currencies list

### LINKED ISSUE #1923: Exchange rate should be listed in enabled currencies list
Enabled currencies list design shows exchange rate in list. This will need to be added.

What about if the merchant has chosen a manual rate?

Screen Shot 2021-05-26 at 10 22 19 PM

### PR DESCRIPTION
**Pull request by luizreis**
Fixes #1923 

### Changes proposed in this Pull Request

This PR adds an "Exchange rate" header and the exchange rate to the enabled currencies list. Based on the designs, the currency exchange rate was rounded to 2 decimal places.

image

### Testing instructions

**Automated tests**

I could not find any existing tests for these components. Should I add some as part of this PR?

**Manual tests**

* Apply this PR and load up the enabled currencies list
* Check that the exchange rate is listed for every currency, except for the default one
* Check that the default currency has a "Default currency" text
* Check that the header and values are aligned and the action section is displayed correctly

-------------------

- [x] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: client/multi-currency/enabled-currencies-list/list-item.js [hunk 1]
```diff
@@ -14,26 +14,33 @@ import DeleteButton from './delete-button';
...
 } ) => {
-	const defaultText = __( 'Default currency', 'woocommerce-payments' );
-	// eslint-disable-next-line camelcase
-	const currencyCode = is_default ? `${ code } - ${ defaultText }` : code;
-
 	const getEditUrl = ( currencyId ) => {
 		return `admin.php?page=wc-settings&tab=wcpay_multi_currency&section=${ currencyId.toLowerCase() }`;
 	};
 
+	const formattedRate = Number.parseFloat( rate ).toFixed( 2 );
```
Comment (luizreis) [inline client/multi-currency/enabled-currencies-list/list-item.js:25]: Should we format it differently when it's a zero-decimal currency? My understanding is that even in those cases the exchange rate could be a float.

Comment (dmallory42) [reply client/multi-currency/enabled-currencies-list/list-item.js:24]: It [seems like](https://www.dailyfx.com/usd-jpy) it's [quite](https://www.xe.com/currencyconverter/convert/?Amount=1&From=JPY&To=USD) common to format the conversion rate as a float even when it is a zero-decimal currency, so I can't see any issue with us doing it to 2dp.

**Comment by dmallory42** [comment]
LGTM and works well! 🚀 I would be happy to either add a few tests as part of this PR, or to make an issue to add the tests and then follow up with another PR on the main feature branch.

**Review by dmallory42 — APPROVED** [review]

**Comment by luizreis** [comment]
> LGTM and works well! 🚀 I would be happy to either add a few tests as part of this PR, or to make an issue to add the tests and then follow up with another PR on the main feature branch.

Thanks for the review! I think we can add the tests as part of https://github.com/Automattic/woocommerce-payments/issues/2026, so I'll merge this.

**Pull request merged**